samples/cpp/beam_search_causal_lm/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+find_package(OpenVINOGenAI REQUIRED PATHS
+    "${CMAKE_BINARY_DIR}"  # Reuse the package from the build.
+    ${OpenVINO_DIR}  # GenAI may be installed alogside OpenVINO.
```

Comment:
@Wovchena 
do we need it now?
`setupvars.sh` exposes `OpenVINOGenAI_DIR`

Reply:
OpenVINO_DIR a good guess anyway.